### DISCUSSION TITLE
Collaboration with VSCODE

### DISCUSSION THREAD

**Discussion by Dabrytskaya** [firity/HYF-GROUP2#26] [General]
I will leave so as not to lose in Slack

1. `git clone` the repo
2. make sure the main branch is updated by `git fetch` `git pul`.
3. `git branch"name"` for new branch
4. `git checkout "name"`
5. Change what you want and save
6. `git commit -a -m"your comment"`
7. `git pull origin main` to check
8. Solve the problems
9. `git push --set-upstream origin "name"`
10. Go to Git and make PR

**Comment by firity** [comment]
@Dabrytskaya  thanks
